static/app/actionCreators/teams.tsx

```diff
@@ -72,6 +75,25 @@ export function joinTeam(
   });
 }
 
+export async function joinTeamPromise(
```

Comment:
useMutation expects a promise, i'll have to clean up joinTeam and rename joinTeamPromise after